app/assets/stylesheets/gradesheet.css.scss

```diff
@@ -36,6 +39,9 @@ td.focus {
   padding-right: 7px;
   white-space: nowrap;
 }
+#grades th.sorting, th.sorting_asc, th.sorting_desc {
```

Comment:
The `#grades` prefix only applies to `th.sorting`. I feel like we should be consistent and either remove `#grades` or add it to all selectors.

Reply:
added to all selectors